freeipa/src/main/resources/freeipa-salt/salt/freeipa/scripts/stageuser.patch

```diff
@@ -0,0 +1,53 @@
+--- stageuser.py.orig	2021-10-28 15:19:12.107391269 +0000
++++ stageuser.py	2021-10-28 15:56:12.973321943 +0000
+@@ -727,26 +727,31 @@
+                              "active %s", active_dn)
+             raise
+ 
+-        # add the user we just created into the default primary group
+-        config = ldap.get_ipa_config()
+-        def_primary_group = config.get('ipadefaultprimarygroup')
+-        group_dn = self.api.Object['group'].get_dn(def_primary_group)
++#        # add the user we just created into the default primary group
```

Comment:
why commenting the lines instead of dropping them?

Reply:
I left them in there during the redhat engagement so that I could find the location of the changes easier. I can remove them.

Reply:
If you commit these patch files into cloudbreak-images repository, then rpms are not required. We need rpms for those things that we consume from a different place, e.g downloading and installing the metering agents etc. requires an rpm.